src/V3EmitCImp.cpp

```diff
@@ -769,7 +769,7 @@ class EmitCTrace final : EmitCFunc {
 
     void emitTraceChangeOne(AstTraceInc* nodep, int arrayindex) {
         iterateAndNextConstNull(nodep->precondsp());
-        const string func = nodep->full() ? "full" : "chg";
+        const std::string func = nodep->traceType() == VTraceType::CHANGE ? "chg" : "full";
```

Comment:
Actually, this is what I was trying to say above. There are 3 types (CONSTANT/FULL/CHG), here both CONSTANT and FULL must print "full", but a comment is warranted.

Reply:
My point was just about refactoring, I think it's better to have this as a function in VTraceType, e.g. changeOneName() or whattever that returns this chg/full.

Reply:
Ah, ok I'm with you, you mean a new function other than `ascii()`. These would only ever be called in one place, as they are all unique. I tend to avoid adding very small function used only once.

Reply:
Either way you want.